pyproject.toml

```diff
@@ -27,12 +27,12 @@ networkx = "^2.7.1"
 types-PyYAML = "^6.0.4"
 nbmake = "^1.3.0"
 rdflib-sqlalchemy = "^0.5.3"
-pyshacl = "^0.25.0"
+pyshacl = ">=0.26,<1.0"
```

Comment:
https://python-poetry.org/docs/dependency-specification/ I think `^0.26` does the same thing. Though it is a bit ambiguous as they don't specify the patch in their versioning...

Reply:
Yeah, I think you're right. Fixed!